app/controllers/task_controller.rb

```diff
@@ -24,18 +38,33 @@ def create
     @task = Task.new(create_params[:task])
     @task.save
 
-    redirect_to "/"
+    redirect_to root_url
   end
 
   def create_params
```

Comment:
I think you have talked about `public` versus `private` methods in ruby classes. For Rails controllers usually people put methods like this, or `sort_name` below, or anything that's not an action in the private section. This is because any public method on a controller can be interpreted by Rails as an action (provided there's a route for it and, possibly a view). So it's safer to make all these private. It also helps group the actions at the top of the code and the rest of the supporting methods below. e.g. something like this:

```ruby
class TasksController < ApplicationController

  def index
    ...
  end

  def show
    ...
  end

  def delete
    ...
  end

  private

  def create_params
     ...
  end
end

```